components/webdriver_server/lib.rs

```diff
@@ -966,10 +966,21 @@ impl Handler {
                     ErrorStatus::UnableToSetCookie,
                     "Unable to set cookie",
                 )),
+                WebDriverCookieError::UnableToDeleteCookies => Err(WebDriverError::new(
+                    ErrorStatus::UnsupportedOperation,
+                    "Unsupported Command For Add Cookie",
+                )),
             },
         }
     }
 
+    fn handle_delete_cookies(&self) -> WebDriverResult<WebDriverResponse> {
+        let (sender, _) = ipc::channel().unwrap();
+        let cmd = WebDriverScriptCommand::DeleteCookies(sender);
+        self.browsing_context_script_command(cmd)?;
```

Comment:
We can check for an error and return https://manishearth.net/sand/doc.servo.org/webdriver/error/enum.ErrorStatus.html#variant.NoSuchWindow.